etc/adminhtml/system/adyen_alternative_payment_methods.xml

```diff
@@ -46,16 +46,5 @@
             <tooltip>This will override the default shopper locale of the Magento store (e.g. nl_NL). Leave empty to let Magento decide.</tooltip>
             <config_path>payment/adyen_hpp/shopper_locale</config_path>
         </field>
-        <field id="cctypes" translate="label" type="multiselect" sortOrder="60" showInDefault="1" showInWebsite="1" showInStore="0">
-            <label>Boleto Types</label>
-            <source_model>Adyen\Payment\Model\Config\Source\BoletoType</source_model>
-            <config_path>payment/adyen_boleto/boletotypes</config_path>
-        </field>
-        <field id="delivery_days" translate="label" type="text" sortOrder="70" showInDefault="1" showInWebsite="1" showInStore="0">
```

Comment:
Don't we need `delivery_days`? Did you double check with the product?

Reply:
After discussion, we are retaining this field